studio/components/interfaces/Auth/Users/Users.tsx

```diff
@@ -70,6 +76,24 @@ const Users = () => {
               ),
             ]}
           />
+          <Listbox
+            size="small"
+            value={PageState.filterVerified}
+            onChange={onVerifiedFilterChange}
+            name="verified"
+            id="verified"
+            icon={<IconUsers size="tiny" />}
```

Comment:
Please remove the IconUsers for now, it breaks the listbox on the longer Un-Verified string.

Reply:
@pucho - Just pushed a commit now to remove the icon.